tracer/src/Datadog.Trace/AppSec/Waf/Waf.cs

```diff
@@ -39,15 +41,25 @@ public Version Version
             }
         }
 
-        public static Waf Initialize()
+        public static Waf Initialize(string rulesFile)
         {
-            var rule = NewRule();
-            return rule == null ? null : new Waf(rule);
+            var wafHandle = CreateWafHandle(rulesFile);
+
+            if (wafHandle == null)
+            {
+                Log.Error("Failed to create rules.");
+            }
+            else
+            {
+                Log.Information("Rules successfully created.");
```

Comment:
Info is logged by default - should we make this debug?

Reply:
+1